ui-vue3/package.json

```diff
@@ -15,6 +15,9 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
     "format": "prettier --write src/"
   },
+  "resolutions": {
+    "jackspeak": "2.1.1"
+  },
```

Comment:
A [workround](https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092) to fix the bug caused by the resolver.